index.js

```diff
@@ -173,20 +173,20 @@ controller.on('ambient', function ambient(bot, message) {
         // validate message
         let valid = containsValidSentences(message.text)
         if (valid < 1) {
-            insultgen(function (insult) {
-                let replyText = `Sorry <@${message.user}>, `;
-                if (valid === 0) {
-                    replyText += `something in your message seems to be not a sentence: ${insult}`;
-                } else {
-                    replyText += `that's not a sentence: ${insult}`;
-                }
-                console.log(replyText);
-                bot.reply(message, {
-                    username: 'bingo',
-                    text: replyText,
-                    icon_emoji: `:anton:`
-                })
+            let insulttext = insult.generate(message.user);
```

Comment:
You need to wrap `message.user` either here or in the `insult.js` into `<@${...}>`. Or is this don here: https://github.com/bpoetzschke/slack-bingo-bot/pull/4/files#diff-99644a4767c92b6b164730a5d91919e5R55

Reply:
Yepp, the second thing is right.